packages/core/package.json

```diff
@@ -39,6 +39,7 @@
     "@jscad/io-utils": "2.0.7",
     "@jscad/modeling": "2.5.2",
     "@jscad/vtree": "2.0.8",
+    "json5": "^2.2.0",
```

Comment:
Let's use a fixed version for json5. That will allow us to control the versions, which may change quickly.

Also, please remove json5 from the WEB package.